simple/msg_sockets.c

```diff
@@ -306,6 +306,13 @@ static int setup_handle(void)
 	struct addrinfo *ai, aihints;
 	const char *bound_addr_str;
 	int ret;
+#if defined(_WIN32)
+	WSADATA data;
+
+	ret = WSAStartup(MAKEWORD(2, 2), &data);
+	if (ret)
+		return HRESULT_FROM_WIN32(ret);
+#endif
```

Comment:
Let's add some sort of ft_init() or ft_startup() call to do this, and have the tests call that instead of adding WSAStartup to each test like this.